src/clj_3df/compiler.cljc

```diff
@@ -87,8 +87,8 @@
                      :bool    boolean?))
 
 (s/def ::predicate '#{<= < > >= = not=})
-(s/def ::aggregation-fn '#{min max count median sum avg variance})
-(s/def ::function '#{truncate})
+(s/def ::aggregation-fn '#{min max count median sum avg})
+(s/def ::function '#{truncate concat})
```

Comment:
Sneaking in a new built-in function... >:(

Reply:
I am slightly sorry ;)